daemon-scheduler/pkg/regex/validate.go

```diff
@@ -0,0 +1,34 @@
+// Copyright 2016 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"). You may
+// not use this file except in compliance with the License. A copy of the
+// License is located at
+//
+//     http://aws.amazon.com/apache2.0/
+//
+// or in the "license" file accompanying this file. This file is distributed
+// on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+// express or implied. See the License for the specific language governing
+// permissions and limitations under the License.
+
+package regex
```

Comment:
I thought about it, but decided to name it 'regex' because we may want to do other things using the regexes other than just validation. For example, in the cluster state service, we use regexes to extract the cluster name from the cluster ARN.